src/ports/server/component.ts

```diff
@@ -251,7 +253,27 @@ export async function createServerComponent({
     app.use(bodyParser.json())
 
     // CORS middleware
-    app.use(cors({ origin: corsOrigin, methods: corsMethods }))
+    // const corsOptions = {
```

Comment:
Would you mind removing this commented code?
By making this change, we need to change the definitions as well, to remove the `regex:` prefix.